packages/client/src/pages/settings/preferences-registry.vue

```diff
@@ -0,0 +1,380 @@
+<template>
+<div class="_formRoot">
+	<div :class="$style.buttons">
+		<MkButton inline class="primary" @click="saveNew">{{ ts._preferencesRegistry.saveNew }}</MkButton>
+		<MkButton inline class="" @click="loadFile">{{ ts._preferencesRegistry.loadFile }}</MkButton>
+	</div>
+
+	<FormSection>
+		<template #label>{{ ts._preferencesRegistry.list }}</template>
+		<div
+			v-if="registries && Object.keys(registries).length > 0"
+			v-for="(registry, id) in registries"
```

Comment:
ESLint is complaining about this:

```
Error:    11:4   error  This 'v-if' should be moved to the wrapper element  vue/no-use-v-if-with-v-for
Error:    12:4   error  Attribute "v-for" should go before "v-if"           vue/attributes-order
```

(one gets solved by the other, if you move `v-if` to the wrapping component)